app/views/admin/responses/index.html.erb

```diff
@@ -1,22 +1,46 @@
-<h1>All responses</h1>
+<div class="title-bar flex flex-h-center flex-v-center gutters">
+  <h1 class="h1--small-uppercase">All survey responses</h1>
+</div>
 
-<table>
-  <thead>
-    <th>ID</th>
-    <th>Created at</th>
-    <th colspan="4"></th>
-  </thead>
-  <tbody>
-    <% @responses.each do |response| %>
-      <tr>
-        <td><%= response.id %></td>
-        <td><%= response.created_at.to_formatted_s(:short) %></td>
-        <td><%= link_to "Show survey page", survey_path(response.survey) %></td>
-        <td><%= link_to "Show results page", results_survey_response_path(response.survey, response) %></td>
-        <td><%= link_to "Show full response", survey_response_path(response.survey, response) %></td>
-        <td><%= link_to 'Destroy', survey_response_path(response.survey, response), method: :delete, data: { confirm: 'Are you sure?' } %></td>
-      </tr>
+<div class="container--medium content-padding">
+
+  <div class="flex flex-column flex-h-center flex-v-center box--grey-light">
+    <p class="text-40"><span class="bold"><%= @responses.count %></span> Total result<% unless @responses.count == 1 %>s<% end %></p>
```

Comment:
You can use pluralize here, just give it the string and the count to pluralize on `<%= "result".pluralize(@responses.count) %>`

Reply:
Alternatively... `<%= pluralize("result", @response.count) %>`, whichever you think is nicer

Reply:
Ah my bad, looks like the arguements are the other way round... https://apidock.com/rails/ActionView/Helpers/TextHelper/pluralize

Reply:
Count first, then string